kedro-org/kedro-viz
# Fix Run Command and Demo Project

Signed-off-by: Rashida Kanchwala <rashida_kanchwala@mckinsey.com>

## Description

This is a small PR that resolves #1225 and #1243 and also adds transcoded datasets to demo-project


## Development notes

- The issue related to MemoryDatasets showing up for different nodes instead of the correct datasets  was because of incorrect naming of the dataset in the catalog.yml file. This has been fixed. 
- I have also added a simple transcoded dataset example for us to test any edge cases related to transcoding on demo project 
- For the run-command issue, all that was required was to add namespace to task nodes when doing the run_command and remove the double quotes. 

Also, currently on kedro-viz, if the user does not specify any 'name' argument for  task node, the run command on metadata panel displays the below error message. 
_'Please provide a name argument for this node in order to see a run command.'_ 

I think this bit can be improved and if there is no name argument, we could possible use the _kedro generated name_ . Currently because of this PR [https://github.com/kedro-org/kedro/pull/2139] it still won't work but once this is released in Kedro 0.18.5 -- we can do the above. 


## QA notes

Open Kedro-viz , expand ingestion and click on any task node. You will see run command show, copy that run_command. Next go to demo-projects on cli and paste the run command and see it run successfully. 

i added an additional transcoded dataset '@ pandas3' just to test the FE. i will remove it before merge as it doesn't make sense for the demo project. 

## Checklist
